src/montecarlo/run_fund_scc_mcs.jl

```diff
@@ -0,0 +1,97 @@
+using Mimi
+
+include(joinpath(@__DIR__, "defmcs.jl"))                  # Defines FUND's distributional parameters
+include(joinpath(@__DIR__, "../new_marginaldamages.jl"))  # Defines FUND's marginal emissions model
+
+"""
+Runs a Monte Carlo simulation with the FUND model over it's distirbutional parameters, and calculates
+the Social Cost of Carbon for the specified `years` and discount `rates`.
+`trials`: the number of trials to run.
+`ntimesteps`: how many timesteps to run.
+`output_dir`: an output directory; if none provided, will create and use "output/SCC yyyy-mm-dd HH-MM-SS MCtrials". 
+`save_trials`: whether or not to generate and save the MC trial values up front to a file.
+"""
+function run_fund_scc_mcs(trials = 10000; years = [2020], rates = [0.03], ntimesteps = 1051, output_dir = nothing, save_trials = false)
+
+    # Set up output directories
+    output_dir = output_dir == nothing ? joinpath(@__DIR__, "../../output/", "SCC $(Dates.format(now(), "yyyy-mm-dd HH-MM-SS")) MC$trials") : output_dir
+    mkpath("$output_dir/results")
+
+    # Write header in SCC output file
+    scc_file = joinpath(output_dir, "scc.csv")
+    open(scc_file, "w") do f 
+        write(f, "trial, rate, year, scc\n")
+    end 
+
+    # Scenario set up
+    scenario_args = [
+        :rate           => rates
+        :emissionyear   => years
+    ]
+
+    mcs = getmcs()
+
+    # Generate trials
+    if save_trials
+        generate_trials!(mcs, trials; filename = joinpath(@__DIR__, "$output_dir/trials.csv"))
+    else 
+        generate_trials!(mcs, trials)
+    end
+
```

Comment:
This highlights the fact that the only difference between the branches is the filename:
```
filename = save_trials ? joinpath(@__DIR__, "$output_dir/trials.csv") : nothing
generate_trials!(mcs, trials; filename=filename)
```